source/backend/dal/Helpers/Extensions/AcquisitionOwnerExtensions.cs

```diff
@@ -0,0 +1,27 @@
+using System.Text;
+using Pims.Dal.Entities;
+
+namespace Pims.Dal.Helpers.Extensions
+{
+    public static class AcquisitionOwnerExtensions
+    {
+        public static string FormatOwnerName(this PimsAcquisitionOwner pimsOwner)
+        {
+            StringBuilder stringBuilder = new();
```

Comment:
Personally I prefer this syntax for conciseness/readibility, what do you think?

string[] names = { person.FirstName, person.MiddleNames, person.Surname };
return string.Join(" ", names.Where(n => n != null && n.Trim() != string.Empty));